backend/dataall/base/api/__init__.py

```diff
@@ -32,6 +32,9 @@ def bootstrap():
         enumclass.toGraphQLEnum()
 
     for cls in classes.keys():
+        if not cls.class_instances:  # if there are no instances of cls registered in the app
+            continue
+
```

Comment:
This is for the case there is no `QueryType` or `UnionType` or any other gql type to register right?

Reply:
Yes, if we don't have `Union` type we should not fail (it can be achieved easily by disabling all modules)